Applications/WebPortal/WebApp/library/source/components/integer_input_box.tsx

```diff
@@ -31,41 +35,38 @@ interface State {
 /** Displays an input box for modifying integer values. */
 export class IntegerInputBox extends React.Component<Properties, State> {
   public static readonly defaultProps = {
+    min: 0,
     value: 0,
     onChange: () => {}
   };
 
   constructor(props: Properties) {
     super(props);
     this.state = {
-      value: props.value || 0
+      value: props.value || this.props.min
     };
     this.onKeyDown = this.onKeyDown.bind(this);
     this.onWheel = this.onWheel.bind(this);
     this.onChange = this.onChange.bind(this);
+    this.onBlur = this.onBlur.bind(this);
   }
 
   public render(): JSX.Element {
-    const value = this.state.value.toString().padStart(
-      this.props.padding || 0, '0');
+    const shownValue = (() => {
```

Comment:
You don't need a lambda expression for this.

Reply:
Should I put it inside value or is it fine where it is?

Reply:
Fine where it is.